WHATS_NEW.md

```diff
@@ -30,7 +34,7 @@ rhoas context unset --services=connector,namespace
 
 ## 0.49.0
 
-### Support for Redhat Openshit Connectors (RHOC)
+### Support for Red Hat Openshift Connectors (RHOC)
 You are now able to interact with RHOC directly from the CLI. You can create a new connector by using `rhoas connector build` in conjunction with `rhoas connector create`.
 
```

Comment:
Should be "Red Hat OpenShift" :)

Suggest dropping "(RHOC)" from the header (internal-only acronym).

Also "You can now interact with Red Hat OpenShift Connectors directly from the CLI."